.travis.yml

```diff
@@ -11,14 +11,26 @@ sudo: false
 
 cache:
   directories:
-    - $HOME/.composer/cache/files
+    - $HOME/.composer/cache
 
 services: mongodb
 
+matrix:
+   include:
+     - php: 5.6
+       env: SYMFONY_VERSION="2.3.*"
+     - php: 5.6
+       env: SYMFONY_VERSION="2.8.*"
```

Comment:
Should we drop 2.3 support for the lib then? Otherwise I would keep the 2.3 build matrix as for this PR there is a difference between 2.3 and 2.7+ optionsresolver